# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/ipc_ping_test.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/ipc_ping_test.cc
--- a/security/sandbox/chromium/sandbox/win/src/ipc_ping_test.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/ipc_ping_test.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -2,22 +2,22 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "testing/gtest/include/gtest/gtest.h"
 #include "sandbox/win/src/sandbox.h"
 #include "sandbox/win/src/sandbox_factory.h"
 #include "sandbox/win/src/target_services.h"
 #include "sandbox/win/tests/common/controller.h"
+#include "testing/gtest/include/gtest/gtest.h"
 
 namespace sandbox {
 
 // Tests that the IPC is working by issuing a special IPC that is not exposed
 // in the public API.
-SBOX_TESTS_COMMAND int IPC_Ping(int argc, wchar_t **argv) {
+SBOX_TESTS_COMMAND int IPC_Ping(int argc, wchar_t** argv) {
   if (argc != 1)
     return SBOX_TEST_FAILED;
 
   TargetServices* ts = SandboxFactory::GetTargetServices();
-  if (NULL == ts)
+  if (!ts)
     return SBOX_TEST_FAILED;
 
   // Downcast because we have internal knowledge of the object returned.